Session_5/Nqueen.java

```diff
@@ -0,0 +1,75 @@
+public class Nqueen {
+
+	int[][] chessBoard;
+
+	/**
+	 * Checks that the specified position is safe or not for placing Queen
+	 * 
+	 * @param board all elements of board are initially initialized to zero
+	 * 
+	 * @param row index of row where queen is placed
+	 * @param column index of column where queen is placed
+	 * @param dimensionOfChessBoard
+	 * @return returns true if specified position is safe or false if specified position is not safe
+	 */
+	public boolean isSafe(int[][] board, int row, int column,
+			int dimensionOfChessBoard) {
+		chessBoard = board;
+		// check if Queen is present in same column
+		for (int count_row = 0; count_row < row; count_row++) {
+			if (chessBoard[count_row][column] == 1) {
+				return false;
+			}
+		}
+		// check if Queen is present in upper left diagonal
+		for (int count_row = row, count_col = column; count_row >= 0 && count_col >= 0; count_row--, count_col--) {
+			if (chessBoard[count_row][count_col] == 1) {
+				return false;
+			}
+		}
+		// check if Queen is present in upper right diagonal
```

Comment:
Avoid multiple return statements

Reply:
changed multiple return statements as 'answer'

Reply:
You haven't changed anything, still, multiple return statement is present. What you made is still introduce a variable.